templates/partials/answers/radio.html

```diff
@@ -1,10 +1,21 @@
 {% from "components/radios/_macro.njk" import onsRadios %}
 
-{{ onsRadios({
+{% set config = {
   "id": answer["id"],
   "name": answer["id"],
   "legend": question_title,
   "legendClasses": "u-vh",
   "radios": map_radio_config(form, answer),
   "error": error
-}) }}
+} %}
+
+{% if 'voluntary' in answer and answer["voluntary"] %}
+  {% set config = config | setAttribute("clearRadios", {
+    "text": _("Clear selection"),
+    "name": "action[clear_radios]",
+    "ariaClearText": "You can clear your answer using the clear selection button after the radio inputs",
+    "ariaClearedText": "You have cleared your answer"
```

Comment:
Needs to be wrapped in `_()` for translation.

Reply:
I don't think any of the aria text is translated, but I'll check with @jrbarnes9